File: app/src/main/java/com/artmurka/artmurkaapp/Model/Pojo/ItemList/AboutGoods/SizePhoto.java

package com.artmurka.artmurkaapp.Model.Pojo.ItemList.AboutGoods;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SizePhoto {

    @SerializedName("thumb")
    @Expose
    private String thumb;
    @SerializedName("photo")
    @Expose
    private String photo;
    @SerializedName("small")
    @Expose
    private String small;

    public SizePhoto(String photo){
        this.photo = photo;
    }

    public String getThumb() {
        return thumb;
    }

    public void setThumb(String thumb) {
        this.thumb = thumb;
    }

    public String getPhoto() {
        return photo;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public String getSmall() {
        return small;
    }

    public void setSmall(String small) {
        this.small = small;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/aboutgoods/SizePhoto.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.aboutgoods

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class SizePhoto(@field:SerializedName("photo")
                @field:Expose
                var photo: String?) {

    @SerializedName("thumb")
    @Expose
    var thumb: String? = null
    @SerializedName("small")
    @Expose
    var small: String? = null

}
